include/envoy/grpc/async_client.h

```diff
@@ -7,6 +7,8 @@
 #include "envoy/grpc/status.h"
 #include "envoy/http/header_map.h"
 
+#include "common/protobuf/descriptor.h"
+
 #include "google/protobuf/descriptor.h"
```

Comment:
This and the rest are not done yet as it's WiP - I'll be cleaning up all these refs, thanks.